### PR TITLE
Home Page Added

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Home Page added for E-Commerce app

- Navbar
- Footer

### Link For Review 
https://e-com-home.netlify.app/

### CODE REVIEW

File: CSS/button.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.btn-primary{
+    background-color: var(--primary-dark-color) !important;
+    font-size: 1.5rem !important;
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color) !important;
+    font-size: 1.5rem !important;
+}
```
Comment (Samarth-5109) [inline CSS/button.css:9]: using "important" is not a good practice, style using specificity instead

File: CSS/button.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    color: var(--secondary-color) !important;
+    background-color: var(--white) !important;
+    font-size: 1.5rem !important;
+
+}
+
+.btn-outline-secondary:hover{
+    background-color: var(--secondary-color) !important;
+    color: var(--white) !important;
+}
```
Comment (prathmeshjagtap) [inline CSS/button.css:34]: you are using to many "important" maybe you can increase the specificity that would work

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]
Everything  Else is fine just work on important CSS property which you have given

File: CSS/home.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    color: var(--primary-dark-color);
+}
+
+#home-banner .banner-content .info{
+    color: var(--primary-dark-color);
+}
+
+/* New Arrivals  */
+
+#new-arrivals{
```
Comment (narharikale) [inline CSS/home.css:55]: it's not a good practice to  use ID selectors

File: CSS/button.css (hunk 1 above)
Comment (Mahi24-prog) [reply CSS/button.css:9]: I used css component library so that to overwrite style I used it

File: CSS/button.css (hunk 2 above)
Comment (Mahi24-prog) [reply CSS/button.css:34]: I used component from css component library so that to overwrite some style I used it

File: CSS/home.css (hunk 3 above)
Comment (Mahi24-prog) [reply CSS/home.css:55]: Ok

**Comment by iamharnad** [comment]
looks good, i think you have made the requested changes!

**Review by iamharnad — APPROVED** [review]

**Pull request merged**